build/Dockerfile

```diff
@@ -86,6 +99,11 @@ LABEL stage=operator
 WORKDIR /payload/
 COPY --from=build /build/windows-machine-config-operator/windows-machine-config-bootstrapper/wmcb.exe .
 
+# Copy containerd.exe and containerd-shim-runhcs-v1.exe
+WORKDIR /payload/containerd/
+COPY --from=build /build/windows-machine-config-operator/containerd/bin/containerd.exe .
+COPY --from=build /build/windows-machine-config-operator/hcsshim/containerd-shim-runhcs-v1.exe .
+
```

Comment:
I might be missing something here, don't you need to change the WORKDIR back to /payload/ after this step, likewise for the other Dockerfile.

Reply:
You are right. That is a good catch. It also tells me that if we are changing the Dockerfile, just ensuring that it builds locally is not sufficient but we need to actually deploy the operator in a dev cluster. My bad for not doing that.

Reply:
I have now fixed this and confirmed that the operator deploys on my dev cluster.